File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int[] indices = new int[2];
        int remainder = 0;

        for(int i = 0 ; i < nums.length ; i++)
        {
            remainder = target - nums[i];

            for(int j = i + 1 ; j < nums.length && i < nums.length ; j++)
            {
                if(remainder == nums[j])
                {
                    indices[0] = i;
                    indices[1] = j;
                }
            
            }
        }
        return indices;
    }
}

File: Data Structures & Algorithms/permutations/submission-1.py
class Solution:
    def permute(self, nums: List[int]) -> List[List[int]]:
        
        perms = []

        def backtrack(candidates, target, current, perms):
            if len(current) >= target:
                perms.append(current[:])
                return
            for cand in candidates:
                if cand not in current:
                    current.append(cand)
                    backtrack(candidates, target, current, perms)
                    current.remove(cand)
            return

        

        backtrack(nums, len(nums), [], perms)

        return perms
        